staging/src/k8s.io/client-go/tools/leaderelection/resourcelock/interface.go

```diff
@@ -100,14 +100,15 @@ type Interface interface {
 }
 
 // Manufacture will create a lock of a given type according to the input parameters
-func New(lockType string, ns string, name string, coreClient corev1.CoreV1Interface, coordinationClient coordinationv1.CoordinationV1Interface, rlc ResourceLockConfig) (Interface, error) {
+func new(lockType string, ns string, name string, coreClient corev1.CoreV1Interface, coordinationClient coordinationv1.CoordinationV1Interface, rlc ResourceLockConfig, labels map[string]string) (Interface, error) {
```

Comment:
I'm confused by the comment above this function.  It is intended to be the godoc for this function?  Should it be moved down to with exported `New` function?